products/health-care/appointments/va-online-scheduling/express-care-research/express-care-project-brief.md

```diff
@@ -1,41 +1,55 @@
-Some Veterans have a care need right now that requires immediate medical attention or guidance. For various reasons, the Veteran may not be able to go into a physical location for this care. Additionally, their care need may not require an in person visit. In both cases, however, the Veteran needs to talk to a medicial professional _today_. 
+Some Veterans have a care need that could be resolved without an in-person appointment and/or requires immediate medical attention or guidance, yet doesn't demand a trip to the ER. For various reasons, the Veteran may not be able to go into a physical location for this care. In both cases, however, the Veteran needs to talk to a medicial professional _today_. 
 
-Currently, "Express Care" is a vehicle/care option by which the VA can provide immediate, same day evaluation via VA Video Connect (or telehealth). 
+Currently, "Express Care" is a vehicle/care option by which the VA can provide immediate, same day evaluation via VA Video Connect (or telehealth appointment). 
 
 Additionally, given the recent COVID-19 pandemic, there is an urgent need to give Veterans fast access to care and evaluation via telehealth either as an alternative to in office, face-to-face visits or as a mechanism for screening patients who suspect they have COVID-19. 
 
 This problem may be a subset to a larger problem statement - Veterans have acess to various types of care which can make it difficult to understand what kind of care to access and how to access it. This is especially true when considering the different VA digital access points to accessing care. 
 
 ## Goal:
-We want to perform user research with various Veterans to collect insights and data on what types of care Veterans would seek same-day care access and how they would go about (care options) receiving this immediate care at the VA with emphasis on understanding use of 'telehealth' or online care options.
+We want to perform usability research with Veterans to learn more about:
+ + Veterans undertanding of what appointment types are available to them through VA, specifically for telemedicine and same-day appointments
+  + how might Veterans seek same-day care (either through VA or other sources)
+  + how might Veterans go about receiving immediate care at the VA, with emphasis on understanding use of 'telehealth' or online care options. 
 
 ## Current Hypothesis:
-We believe the existing vehicle for delivering this same day care "Express Care" online is a separate flow from VA Online Schedule because urgent online consultation/care is same day and not future facing like VA Online Scheduling. 
+We believe there are two major hinderances to Express Care as a service for urgent/same-day appointments for Veteran patients:
+ + Veterans can only see their PACT, if and *only if* their PACT makes availability during normal business hours
+ + Veterans are unaware of Express Care as an option for scheduling an appointment, due to how appointment scheduling is routed. 
```

Comment:
+ Veterans are unaware of Express Care as an option for scheduling an appointment, due to unavailability from providers
+ Veterans won't think to make an Express Care appointment the same way as other types of VA appointments because of the timeliness of the appointment